### PR TITLE
fix-6617

### PR DESCRIPTION
**Pull request by pallavi-pannu-oc**

### CODE REVIEW

File: dkube/sdk/internal/api_base.py [hunk 1]
```diff
@@ -97,8 +97,15 @@ def list_runs(self, category, user, shared=False, filters='*'):
         # inference runs
         if category == 'inference':
             all = 'false'
-        else:
-            all = 'true'
+        if category == 'inference':
```
Comment (songole) [inline dkube/sdk/internal/api_base.py:100]: Do you ever set it to true? Can all be set to false all the time?

Comment (pallavi-pannu-oc) [reply dkube/sdk/internal/api_base.py:100]: No @songole , if in future some other category comes, then we might have to add. for now i will set to true all at once.

**Review by songole — APPROVED** [review]

**Comment by hemanthravi** [comment]
@songole 
Is the above comment an issue for this PR. Does this need to be reverted?

File: dkube/sdk/internal/api_base.py (hunk 1 above)
Comment (songole) [reply dkube/sdk/internal/api_base.py]: @pallavi-pannu-oc should it be set to true or false?

**Comment by songole** [comment]
@hemanthravi which comment are you referring to?

**Pull request merged**